### ISSUE TITLE
Requiring unknown module "react-native-scrollable-tab-view"

### ISSUE THREAD

**Issue by chenEdgar** [skv-headless/react-native-scrollable-tab-view#492]
system: win8.1
platform: android
dependency: 
react-native: 0.37.0 / 0.39.1
react-native-cli: 1.3.0
react-native-scrollable-tab-view: 0.7.0

in react packager, I saw this Error tips:
 `TypeError: Cannot convert undefined or null to object
    at Function.from (native)
    at _toArray (E:\react\gogo\node_modules\react-native\local-cli\server\util\l
aunchEditor.js:9:78)
    at launchEditor (E:/react/gogo/node_modules/react-native/local-cli/server/ut
il/launchEditor.js:142:59)
    at Object.handle (E:/react/gogo/node_modules/react-native/local-cli/server/m
iddleware/openStackFrameInEditorMiddleware.js:17:7)
    at next (E:\react\gogo\node_modules\connect\lib\proto.js:174:15)
    at Object.handle (E:/react/gogo/node_modules/react-native/local-cli/server/m
iddleware/getDevToolsMiddleware.js:74:7)
    at next (E:\react\gogo\node_modules\connect\lib\proto.js:174:15)
    at Object.handle (E:/react/gogo/node_modules/react-native/local-cli/server/m
iddleware/getDevToolsMiddleware.js:74:7)
    at next (E:\react\gogo\node_modules\connect\lib\proto.js:174:15)
    at Object.compression [as handle] (E:\react\gogo\node_modules\compression\in
dex.js:205:5)`

I tried to reinit a project， reinstall module "react-native-scrollable-tab-view", and rerun the project. but failed.

**Comment by skv-headless** [comment]
I don't have windows machine. Have no idea what is the problem.

**Comment by chenEdgar** [comment]
I am puzzled now.
I uninstalled the module of "react-native-scrollable-tab-view" and "react-native-tab-navigator" using `npm uninstall <pkg>`.
then I used `yarn add react-native-tab-navigator`, then I saw this.
![image](https://cloud.githubusercontent.com/assets/12596622/21078909/79f3cf78-bfbb-11e6-9b36-347e0a6a8848.png)
and then I can use both module in my project.
 so may be the answer of the question is that mixed using `npm` and `yarn` command in the same project may caused some problem? I really don't know.
